packages/adblocker/src/filters/network.ts

```diff
@@ -1535,6 +1554,14 @@ export default class NetworkFilter implements IFilter {
       options.push('badfilter');
     }
 
+    if (this.isRemoveParam()) {
+      if (this.removeparam!.length > 0) {
```

Comment:
wont this throw if `this.removeparam === undefined` ?

Reply:
`removeparam` is always `string` here. I'll add some comments or improve the type clarity.

Reply:
shouldn't this type change then?
```
public get removeparam(): string | undefined {
```
https://github.com/ghostery/adblocker/pull/4528/files#diff-f45592ce8a6bd8a6e8e39391572f4a38406614b3649a6628891ca6fc38a516a9R1279

Reply:
- Other methods like `get csp()` has a same shape.
- We use an empty value for the full removal of parameters.